src/components/common/Header/NavContent.jsx

```diff
@@ -1,12 +1,19 @@
+import Link from 'next/link'
 import styles from '@/styles/common/Header/NavContent.module.scss'
 
-const NAV_ITEMS = ['XXXXX', 'XXXXX', 'XXXXX']
+const NAV_ITEMS = ['About']
 const NavContent = () => {
   return NAV_ITEMS.map((item, idx) => (
-    <li className={styles['nav-content']} key={idx}>
-      <a className={styles['nav-content-link']} href={`#${item}`}>
+    <li
+      className={styles['nav-content']}
+      key={idx}
+    >
+      <Link
+        className={styles['nav-content-link']}
+        href={item.toLowerCase()}
```

Comment:
```suggestion
        href={`/${item.toLowerCase()}`}
```
I found a navigation bug here.
The file path should start with '/'